gae/room/ndb_models.py

```diff
@@ -1195,9 +1210,15 @@ class StaffPosition(SearchableModel):
   hourly_rate_after_date = ndb.StringProperty(repeated=True)
   mileage_rate_after_date = ndb.StringProperty(repeated=True)
 
-  last_editor = ndb.UserProperty()
+  rates = ndb.StructuredProperty(RateAfterDate, repeated=True)
```

Comment:
Don't need to use a StructuredProperty in the database level.  Using the strings connected with " " is fine.